sklearn/cross_decomposition/_pls.py

```diff
@@ -581,7 +582,13 @@ class PLSRegression(_PLS):
 
     coef_ : ndarray of shape (n_features, n_targets)
         The coefficients of the linear model such that `Y` is approximated as
-        `Y = X @ coef_`.
+        `Y = X @ coef_ + intercept_`.
```

Comment:
In the future when `coef_` becomes `n_targets, n_features` we need to remember to update these docstrings. I guess the best place to put the reminder is here?

https://github.com/scikit-learn/scikit-learn/blob/9752c03bf169dcbdfcc9fa387544e4fb15ddfb32/sklearn/cross_decomposition/_pls.py#L499-L503